app/models/list.rb

```diff
@@ -0,0 +1,12 @@
+class List < ApplicationRecord
+  has_many :subscriptions, dependent: :destroy
+  has_many :contacts, through: :subscriptions
+
+  def subscribe(subscription_id)
+    SubscribeToListJob.perform_later(subscription_id)
+  end
```

Comment:
Haha, this is how I had it and decided to put it on the model in case you needed to interact with the list subscriptions elsewhere. I'll change it back though, I have to reason for it here.